tools/accuracy_checker/accuracy_checker/adapters/text_recognition.py

```diff
@@ -220,6 +221,52 @@ def decode(prob_index, blank_id):
         return selected_index
 
 
+class SimpleDecoder(Adapter):
```

Comment:
I think need to add adapter to README and __init__.py

Reply:
Added.